src/pages/sponsors/edit-sponsor-page.js

```diff
@@ -39,88 +40,169 @@ import {
-          <SponsorForm
-            history={history}
-            entity={entity}
-            currentSummit={currentSummit}
-            sponsorships={sponsorships}
-            errors={errors}
-            onCreateCompany={this.props.createCompany}
-            onAttachImage={this.props.attachSponsorImage}
-            onRemoveImage={this.props.removeSponsorImage}
-            onAddMember={this.props.addMemberToSponsor}
-            onRemoveMember={this.props.removeMemberFromSponsor}
-            onSponsorAdsOrderUpdate={this.props.updateSponsorAdsOrder}
-            onAdvertisementDelete={this.props.deleteSponsorAdvertisement}
-            onSponsorMaterialOrderUpdate={this.props.updateSponsorMaterialOrder}
-            onMaterialDelete={this.props.deleteSponsorMaterial}
-            onSocialNetworkDelete={this.props.deleteSponsorSocialNetwork}
-            onSubmit={this.props.saveSponsor}
-            getSponsorAdvertisements={this.props.getSponsorAdvertisements}
-            getSponsorMaterials={this.props.getSponsorMaterials}
-            getSponsorSocialNetworks={this.props.getSponsorSocialNetworks}
-            canEditSponsors={canEditSponsors}
-            canEditSponsorExtraQuestions={canEditSponsorExtraQuestions}
-            canEditLeadReportSettings={canEditLeadReportSettings}
-            deleteExtraQuestion={this.props.deleteExtraQuestion}
-            updateExtraQuestionOrder={this.props.updateExtraQuestionOrder}
-            availableLeadReportColumns={entity.available_lead_report_columns}
-            upsertSponsorLeadReportSettings={
-              this.props.upsertSponsorLeadReportSettings
-            }
-          />
-        )}
-      </div>
-    );
-  }
-}
+  return (
+    <Box>
+      <Container maxWidth="lg">
+        <Typography fontSize="3.4rem" variant="h4">
+          {entity.company?.name}
+        </Typography>
+        <Box sx={{ borderBottom: 1, borderColor: "divider", mt: 2 }}>
+          <Tabs
+            value={selectedTab}
+            onChange={handleTabChange}
+            aria-label="basic tabs example"
+            sx={{
+              minHeight: "36px"
+            }}
+          >
+            {tabs.map((t) => (
+              <Tab
+                key={t.value}
+                label={t.label}
+                value={t.value}
+                sx={{
+                  fontSize: "1.4rem",
+                  lineHeight: "1.8rem",
+                  height: "36px",
+                  minHeight: "36px",
+                  px: 2,
+                  py: 1
+                }}
+                {...a11yProps(t.value)}
+              />
+            ))}
+          </Tabs>
+        </Box>
+      </Container>
+      <CustomTabPanel value={selectedTab} index={0}>
+        <div className="container">
+          <h3>
+            {title} {T.translate("edit_sponsor.sponsor")}
+            <AddNewButton entity={entity} />
+          </h3>
+          <hr />
+          {currentSummit && (
+            <SponsorForm
```

Comment:
remove this .. we are not using it anymore

Reply:
<img width="2499" height="1814" alt="image" src="https://github.com/user-attachments/assets/c89ee0d4-975a-4b46-8e5a-cb16bd390f8e" />